dist/README.md

```diff
@@ -0,0 +1,41 @@
+# Grist Cloud Distribution
+
+This a comprehensiveversatile distribution of Grist made with Docker Compose. It
```

Comment:
Did you intend to make this neologism or is it a typo?

Reply:
At first I thought it was versatile, then I realise maybe it wasn't, and I left the edit mistake in there.